lxd/vm_qemu.go

```diff
@@ -1022,28 +1040,66 @@ bootindex = "1"
 	return nil
 }
 
+// addConfDriveConfig adds the qemu config required for adding the config drive.
 func (vm *vmQemu) addConfDriveConfig(sb *strings.Builder) {
 	sb.WriteString(fmt.Sprintf(`
 # Config drive
-[fsdev "qemu_config"]
+[fsdev "lxd_config"]
```

Comment:
I thought we were using `qemu_` for devices that aren't LXD devices and `lxd_NAME` for devices that are in the container config.

So you'd get `lxd_root`, `lxd_config` (if the config drive device is named `config`), ... but anything internal can't be in the `lxd_` namespace or we may collide.

Reply:
@stgraber I wasn't aware that was the naming scheme, I just thought it was inconsistent (hence my attempt to improve it).

What constitutes an LXD device? Something specified in the config of the VM?